instructionAPI/src/syscalls.C

```diff
@@ -179,6 +179,15 @@ namespace aarch64 {
   }
 }
 
+namespace riscv64 {
+  bool isSystemCall(di::Instruction const& ins) {
+    auto const id = ins.getOperation().getID();
+
+    return id == riscv64_op_ecall;
```

Comment:
What about ebreak?

Reply:
Did you have a chance to check on this one?

The RISC-V Instruction Set Manual Volume I
Version 20240411
2.8. Environment Call and Breakpoints